### ISSUE TITLE
Feature Request: Show app placement in review line

### ISSUE THREAD

**Issue by ChristopherNugent** [elementary/appcenter-dashboard#92]
It would be nice if the AppCenter dashboard would show the position the app in the review line, to give developers on estimate on how long the review process will take. It might also show an estimate of how long it will take before the application is reviewed.

**Comment by danrabbit** [comment]
Gonna move this to AppCenter Dashboard. Since we now have reviews as PRs, I wonder if we can do something with GitHub API here? At least we could show the number of apps currently in the queue or I dunno

**Comment by cassidyjames** [comment]
@danrabbit I'm mixed on this because you can also just click through to see the PRs themselves to get a sense of the review queue; and since it's not a strictly linear process, that might be a more honest look at the queue than trying to deduce that on the dashboard?

**Comment by danrabbit** [comment]
Yeah, maybe then just a link that's like "10 apps in the review queue" or something?
